### PR TITLE
Add subsetter, add dottedCircle, drop kerning < 20 units

### PR DESCRIPTION
**Pull request by madig**
* Generate dottedCircle (U+25CC) glyph unless if exists already in the sources
* Change metadata fixer to drop kerning values below absolute 20 fU
* Hook up subsetter from GS to clean up debris

@MariannaPaszkowska @chrissimpkins

### CODE REVIEW

**Comment by chrissimpkins** [comment]
@dberlow @sannorozco 

We introduced automated removal of all kerns < 20 units in a 2000 UPM as of this PR. Please let us know if your opinion changes about this.

**Comment by chrissimpkins** [comment]
@madig did you do any testing of the subsetter addition font binary diff on your side?

File: scripts/prune_font_binary.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    logging.basicConfig(format="%(levelname)s: %(message)s", level=logging.WARNING)
+    LOGGER = logging.getLogger(__name__)
+
+    for font in parsed_args.fonts:
+        local_filepath = font.resolve()
+        local_filepath_subset = f"{local_filepath}.subset"
+
+        # The subsetter configuration preserves all OT feature support.
+        # It will remove unused, unencoded glyphs.
+        subset_args_expert = [
```
Comment (chrissimpkins) [inline scripts/prune_font_binary.py:39]: Do we feel confident these subsetter options are sufficient and all appropriate for GS Flex?

**Comment by madig** [comment]
I didn't test it because it's been running fine on GS.

File: scripts/prune_font_binary.py (hunk 1 above)
Comment (madig) [reply scripts/prune_font_binary.py:39]: They strike me as unremarkable.

**Pull request merged**